cartoframes/dataset.py

```diff
@@ -91,6 +123,18 @@ def exists(self):
             self.cc._debug_print(err=err)
             return False
 
+    def get_data(self):
+        if self.type == Dataset.TABLE_TYPE:
+            return 'SELECT * FROM "{schema}"."{table_name}"'.format(schema=self.schema, table_name=self.data)
```

Comment:
I'd expect the `get_data` method to return the actual data not a query.

My suggestion would be creating a `get_query` and `get_data` methods, and do our best to return a `query` or a `dataframe` in any case.

Reply:
We have changed (@oleurud and me) the Dataset class, to everything related to type and data ahs changed.